src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -919,7 +919,7 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
 
             if (_terminal->IsCopyOnSelectActive())
             {
-                CopySelectionToClipboard(!shiftEnabled);
+                CopySelectionToClipboard(shiftEnabled);

```

Comment:
how did this get backwards anyway?

Reply:
So what happened here was that the keybinding deserializer for `copy` was creating a `copy, trimWhitespace=false`?